core/templates/dev/head/pages/library/Library.js

```diff
@@ -250,19 +250,18 @@ oppia.controller('Library', [
     };
 
     // The carousels do not work when the width is 1 card long, so we need to
-    // handle this case discretely.
-    // Also preventing swiping past the first and last card.
+    // handle this case discretely. Also preventing swiping past the first
+    // and last card.
     $scope.incrementLeftmostCardIndex = function(ind) {
       var lastItem = ($scope.libraryGroups[ind].activity_summary_dicts.length -
```

Comment:
Let's wrap the outer expression with parenthesis so it looks like:
```js
var lastItem = (($scope.libraryGroups[ind].activity_summary_dicts.length -
  $scope.tileDisplayCount) <= $scope.leftmostCardIndices[ind]);

```

Reply:
Done.